### PR TITLE
Unify ccce expenses flow - Fixes-1

### PR DESCRIPTION
**Pull request by Dimple16**
Screenshot 2022-03-28 at 2 45 54 PM
Screenshot 2022-03-28 at 2 46 30 PM
Screenshot 2022-03-28 at 2 46 56 PM

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.html [hunk 1]
```diff
@@ -1025,7 +1035,7 @@
 
       <ng-container *ngIf="etxn$|async as etxn">
         <ng-container
-          *ngIf="(etxn && etxn.tx.creator_id === 'SYSTEM_CORPORATE_CARD' && selectedCCCTransaction) && !(isUnifyCcceExpensesSettingsEnabled && isCccExpense)"
+          *ngIf="(etxn && etxn.tx.creator_id === 'SYSTEM_CORPORATE_CARD' && selectedCCCTransaction) && !isUnifyCcceExpensesSettingsEnabled"
```
Comment (saitejasFyle) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.html:1038]: Why isCCCExpense has been removed?

File: src/app/fyle/view-expense/view-expense.page.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
         <div *ngIf="isProjectShown" class="view-expense--card-container__project">
           {{ projectFieldName | titlecase }}: {{ etxn.tx_project_name ? etxn.tx_project_name : 'Unspecified' }}
         </div>
-        <div *ngIf="isUnifyCcceExpensesSettingsEnabled && isCCCTransaction">
+        <div *ngIf="isUnifyCcceExpensesSettingsEnabled && isCCCTransaction && cardNumber">
```
Comment (saitejasFyle) [inline src/app/fyle/view-expense/view-expense.page.html:68]: What if card number is not present - Don't we have to display anything?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.html (hunk 1 above)
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.html:1038]: `selectedCCCTransaction` serves the same purpose, so removed the redundant check

File: src/app/fyle/view-expense/view-expense.page.html (hunk 2 above)
Comment (Dimple16) [reply src/app/fyle/view-expense/view-expense.page.html:68]: this we need confirmation from design, as of now, showing only the label looked odd, so we'll be showing this section only if it's present

**Pull request merged**